fastlane/lib/assets/DefaultFastfileTemplate

```diff
@@ -69,5 +69,5 @@ end
 # More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
 # All available actions: https://docs.fastlane.tools/actions
 
-# fastlane reports which actions are used
-# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer
+# fastlane reports which actions are used. No personal data is recorded. 
+# Learn more at https://github.com/fastlane/fastlane/tree/master/fastlane#statistics
```

Comment:
Shouldn't also https://github.com/fastlane/fastlane#statistics work? (and all other occurrences)